ext/compressors/iaa/iaacodec/src/IAACompressionCodecDeflate.cpp

```diff
@@ -180,6 +180,10 @@ SoftwareCodecDeflate::doCompressData(
     job_ptr->flags = QPL_FLAG_FIRST | QPL_FLAG_DYNAMIC_HUFFMAN | QPL_FLAG_GZIP_MODE |
       QPL_FLAG_LAST | QPL_FLAG_OMIT_VERIFY;
 
+    // Touch dest buffer in advance to avoid page fault on it triggered by accelerator.
+    for(uint32_t p = 0; p < dest_size; p += 4096)
```

Comment:
The 4096 value chosen assumes the page size is 4k? I think we should read the configured page size instead of blindly use 4096.

Reply:
We can extract this code to a separate function and avoid duplicating the code.

Reply:
Thanks korteland
we are preparing for a new jira and contributor license agreement.